bittensor/_subtensor/subtensor_impl.py

```diff
@@ -729,6 +762,197 @@ def add_stake(
         
         return False
 
+    def add_stake_multiple (
```

Comment:
Seems to have some overlapping function with add_stake, maybe we can generalise them tgt later...

Reply:
Yea, it was like that before, we can potentially overload the single stake or call the multiple stake from the single